README.md

```diff
@@ -82,6 +83,33 @@ Typed doubles rely on the type signatures being defined. What if you don't have
 
 2) Auto-generating types on-the-fly from the real call traces (see below).
 
+### Using with Sorbet
+
+To use MockSuey with Sorbet, configure it as follows:
+
+```ruby
+MockSuey.configure do |config|
+  config.type_check = :sorbet
+end
+```
+
+Make sure that `sorbet` and `sorbet-runtime` gem are present in the bundle according to the [sorbet instruction](https://sorbet.org/docs/adopting#step-1-install-dependencies).
+That's it! Now all mocked methods are type-checked.
+
+### raise_on_missing_types
+
+Gem `sorbet-runtime` does not load signatures for stdlib types (Integer, String, etc...) into runtime.
+Checking types for Integer, String, etc is only available through `rbs typecheck` command which uses [custom ruby binary](https://github.com/sorbet/sorbet/blob/master/docs/running-compiled-code.md).
+
+Therefore, you should consider changing `raise_on_missing_types` to `false` if you use Sorbet.
```

Comment:
Maybe, we can somehow distinguish stdlib types from custom types, so we can ignore only missing stdlib types? Any ideas?

Reply:
Thanks for the suggestion, it should be easy to implement.

```ruby
# sorbet.rb
def sig_is_missing(method_call, raise_on_missing)
  return unless raise_on_missing
  return unless method_call.receiver_class < T::Sig # By adding this line to check if any class uses signatures
  raise MissingSignature, RAISE_ON_MISSING_MESSAGE
end
```


I should also mention that I missed an important part of the problem here and I should update the readme to include this information.

If a signature is described in a `.rb` file, it will be used by `sorbet-runtime` and type checking will be available. One of the gems that is using sorbet signatures is [ShopifyAPI](https://github.com/Shopify/shopify-api-ruby/blob/main/lib/shopify_api/auth.rb) for example.

However, many signatures are declared inside `.rbi` files, like 1) signatures for [stdlib and core types](https://github.com/sorbet/sorbet/tree/master/rbi/core) and 2) signatures for most libraries including [rails](https://github.com/chanzuckerberg/sorbet-rails/blob/master/sorbet/rbi/sorbet-typed/lib/activerecord/all/activerecord.rbi). Unfortunately, these types cannot be loaded into runtime at the moment. Therefore, it's not possible to type check their mocks yet.

Reply:
I also updated readme to make the explanation more understandable.
[link to the comment](https://github.com/iurev/mock-suey/commit/09e9a5e218a47628a19168b2fb824e27bd02b98f)

I can add this commit to the `master` branch too.